packages/pygments-extension/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@jupyterlab/pygments-extension",
+  "version": "4.0.0-alpha.5",
+  "description": "Pygments highlighting using the JupyterLab CSS variables",
+  "homepage": "https://github.com/jupyterlab/jupyterlab",
+  "bugs": {
+    "url": "https://github.com/jupyterlab/jupyterlab/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jupyterlab/jupyterlab.git"
+  },
+  "license": "BSD-3-Clause",
+  "author": "Project Jupyter",
+  "sideEffects": [
+    "style/*.css",
+    "style/index.js"
+  ],
+  "style": "style/index.css",
+  "files": [
+    "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "scripts": {},
```

Let's add build and clean scripts

> We need to add `jupyterlab_pygments` and `pygments` as Python dev dependencies as they are needed by the generator scripts

```suggestion
  "scripts": {
    "build": "python generate_css.py",
    "build:all": "npm run build",
    "clean": "rimraf style/base.css",
  },
```

---

That will make jupyterlab dependent on jupyterlab_pygments and pygments at build time